runtime.go

```diff
@@ -190,7 +189,7 @@ func (r *runtime) CompileModule(ctx context.Context, binary []byte) (CompiledMod
 		return nil, errors.New("binary == nil")
 	}
 
-	if len(binary) < 4 || !bytes.Equal(binary[0:4], binaryformat.Magic) {
+	if len(binary) < 4 {
```

Comment:
I think you can take this entire if statement out, it is from when we used to look for the text format.

Reply:
Thank you for review. Fixed